assembl/tests/fixtures/graphql.py

```diff
@@ -446,16 +458,40 @@ def create_proposal_x(graphql_request, first_question_id, idx):
         }
     }
 }
-""" % (first_question_id, idx), context_value=graphql_request)
+""" % (first_question_id, body, publication_state),
+        context_value=graphql_request)
     return res.data['createPost']['post']['id']
 
 
 @pytest.fixture(scope="function")
-def proposals(graphql_request, thematic_and_question):
+def proposals(graphql_request, admin_user, thematic_and_question):
+    from assembl.models import PublicationStates
+    graphql_request.authenticated_userid = admin_user.id
     thematic_id, first_question_id = thematic_and_question
     proposals = []
-    for idx in range(15):
-        proposal_id = create_proposal_x(graphql_request, first_question_id, idx)
+    for idx in range(20):
```

Comment:
There is a test requiring exactly 15 proposals. If you change this fixture, please adapt the current tests or create another fixture.

Reply:
Again I don't remember why Cédric changed this. Do you think I should put it back to 15 or do you see the relevance of 20 ?

Reply:
fixed in https://github.com/assembl/assembl/pull/637/commits/829ecb0d9fb051b7d3f0951df7cc4ba250fbb8f7